docker/importer/importer.py

```diff
@@ -55,6 +55,16 @@ def _is_vulnerability_file(source_repo, file_path):
   return file_path.endswith(source_repo.extension)
 
 
+def aestnow() -> datetime:
+  """Get the current AEST time"""
+  # - First make datetime timezone aware
+  # - Then set timezone to AEST without DST (+10)
+  # - Then make datetime timezone unaware again to keep
+  #   compatibility with ndb
+  return utcnow().replace(tzinfo=datetime.timezone.utc).astimezone(
+      datetime.timezone(datetime.timedelta(hours=11))).replace(tzinfo=None)
```

Comment:
Can you explain in the comment why this is 11 when the comment says +10 ??

Reply:
Oops, that should be 10, thought I fixed it earlier, but didn't commit it properly,